Makefile

```diff
@@ -80,6 +81,15 @@ ifeq ($(GOOS),darwin)
 	codesign --force --deep --sign - $@
 endif
 
+$(BUILD_DIR)/$(PROJECT): $(EMBEDDED_FILES_CHECK_V2) $(GO_FILES) $(BUILD_DIR)
+	$(eval ldflags = $(GO_LDFLAGS) $(patsubst %,-extldflags \"%\",$(LDFLAGS_$(GOOS))))
+	$(eval tags = $(GO_BUILD_TAGS_$(GOOS)) $(GO_BUILD_TAGS_$(GOOS)_$(GOARCH)))
+	GOOS=$(GOOS) GOARCH=$(GOARCH) CGO_ENABLED=1 \
+	    go build -gcflags="all=-N -l" -tags "$(tags)" -ldflags "$(ldflags)" -o $@ $(BUILD_PACKAGE)
+ifeq ($(GOOS),darwin)
+	codesign --force --deep --sign - $@
+endif
```

Comment:
introduction of this target creates warnings when trying to run `make out/skaffold`

Reply:
Ah yes, I see this now as well.  There are actually warnings for each make command attempted.  Do you know what this check does?  I'm currently trying to find the best way to add a similar thing for the v2 fork, will update the thread here when I find a solution

Reply:
Was able to fix this, marking resolved